src/Config/Config.sample.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace NYPL\HoldRequestResultConsumer\Config;
+
+class Config
+{
+    const API_BASE_URL = 'https://api.nypltech.org/api/v0.1';
```

Comment:
Good call! Yeah, I think for security and privacy, it'll make more sense to make them environment variables. Thanks!